modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,228 @@
+
+image::shared:iceberg-integration.png[]
+
+When you enable the Iceberg integration for a Redpanda topic, Redpanda brokers store streaming data in the Iceberg-compatible format in Parquet files in object storage. The brokers also write events to segment files stored on disk. Storing the streaming data in Iceberg tables in the cloud allows you to derive real-time insights through compatible business intelligence and machine learning tools. 
+
+== Enable Iceberg integration
+
+To use the Iceberg integration for a topic, you must set the cluster configuration property `iceberg_enabled` to `true`, and also configure the topic property `redpanda.iceberg.mode`. You can choose to provide a schema if you need the Iceberg table to be structured with defined columns.
+
+. Set the `iceberg_enabled` configuration option on your cluster to `true`. 
++
+[,bash]
+----
+rpk cluster config set enable_datalake true 
+----
++
+[,bash,role=no-copy]
+----
+Successfully updated configuration. New configuration version is 2.
+----
+
+. Create a new topic.
++
+[,bash,]
+----
+rpk topic create <new-topic-name> --partitions 1 --replicas 1
+----
++
+[,bash,role=no-copy]
+----
+TOPIC            STATUS
+new-topic-name   OK
+----
+
+. Enable the integration for the topic by configuring `redpanda.iceberg.mode`. You can choose one of the following modes:
++
+. `KEY_VALUE`: Creates an Iceberg table with a `Key` column and a `Value` column. Redpanda stores the raw topic content in the `Value` column.
+. `VALUE_SCHEMA_ID_PREFIX`: Creates an Iceberg table whose structure matches the Redpanda schema for this topic, with columns corresponding to each field. Redpanda parses the topic values per field and stores them in the corresponding table columns. 
+. `DISABLED` (default): Disables the creation of an Iceberg table for this topic.
++
+[,bash]
+----
+rpk topic alter-config <new-topic-name> --set redpanda.iceberg.mode=<topic-iceberg-mode>
+----
++
+[,bash,role=no-copy]
+----
+TOPIC            STATUS
+new-topic-name   OK
+----
+
+. Register a schema for the topic (optional). You must also set the `redpanda.iceberg.mode` topic property to `VALUE_SCHEMA_ID_PREFIX`. Redpanda uses xref:manage:schema-reg/schema-id-validation.adoc#set-subject-name-strategy-per-topic[`TopicNameStrategy`] (`<topic-name>-value` or `<topic-name>-key`) to find the matching schema based on the subject name. Redpanda derives the Iceberg table format from the schema and writes data files to object storage in this format. Protobuf and Avro schemas are supported.
```

Comment:
We also need to point out that this mode relies on the use of [SerDes](https://docs.confluent.io/platform/current/schema-registry/fundamentals/serdes-develop/index.html#supported-formats) and the  [wire format](https://docs.confluent.io/platform/current/schema-registry/fundamentals/serdes-develop/index.html#wire-format) to provide the schema ID and magic byte in the header of records.

@rockwotj check my exact math here please